roles/config-docker/tasks/kernel_cgroups_v1.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Update Grub config file
+  shell: grubby --args="{{ grub_add_arg }}" --update-kernel=ALL
+
+- name: Reboot the host
```

Comment:
This will "always" reboot the host as written, even if the above grubby line didn't change anything (i.e.: the host already uses v1). A good follow-up would be to make the reboot happen only if a change was made. Not sure if `grubby` will provide a returncode that can be used for this, but another way would be to use the ansible lineinfile module instead - that would also help with the fact that `grubby` isn't always installed (which should be another enhancement if we continue to use `grubby` - i.e.: ensure it's installed before using it).